### ISSUE TITLE
How to create a group chat?

### ISSUE THREAD

**Issue by SixDigitCode** [iandwelker/smserver#133]
Sorry if there is an obvious answer to this question, but is there a way to create a group chat from SMServer? #118 seems to suggest that it's possible, but I can't figure it out (either in the web GUI or the API). I tried sending a new message to `+12002345678, +12009876543`, but that just gives me a "Message Send Failure" notification on the iPhone. Any help would be appreciated!

I'm using SMServer 0.8.0 on an (original) iPhone SE running iOS 14.4.2 and Checkra1n.

**Comment by iandwelker** [comment]
Yeah, it's really funky that it doesn't work now; I tried out a few things on my own and also couldn't get the phone to create a group chat no matter how I formatted the address, so there's definitely an issue.

For the record, SMServer has no officially supported method of creating group chats; it doesn't like check to see if there are address delimiters and then split them and do something different, but in the past it seemed that the private iMessage frameworks would automatically handle that and create group chats if you split the addresses with commas (or something like that). I doubt that anything changed within these frameworks, though (since I've also been on 14.4 since like march), so idk why it's not working now.

I guess I'll have to look into adding support within SMServer itself to do this. I'll update this issue once the API has official support for it.
